.github/workflows/ci.yml

```diff
@@ -262,11 +262,28 @@ jobs:
 
       - name: "Install docs build requirements"
         run: |
+          pip uninstall ansys-sphinx-theme

```

Comment:
is this needed??

Reply:
It was to make sure I was installing the branch version. I will have it removed once `ansys-sphinx-theme` is released.